crates/rpc/rpc-types-compat/src/transaction/mod.rs

```diff
@@ -189,20 +190,21 @@ pub fn transaction_to_call_request(tx: TransactionSignedEcRecovered) -> CallRequ
     };
     let max_priority_fee_per_gas = tx.transaction.max_priority_fee_per_gas();
 
-    CallRequest {
+    TransactionRequest {
         from: Some(from),
         to,
-        gas_price: gas_price.map(U256::from),
-        max_fee_per_gas: max_fee_per_gas.map(U256::from),
-        max_priority_fee_per_gas: max_priority_fee_per_gas.map(U256::from),
+        gas_price: gas_price.map(U128::from),
```

Comment:
now that I see that I think the rpc object should also use u256 because it's more common

I assume, u256 is more widely used

Reply:
agree this is why i stopped porting, going to change it to u256 before continuing this pr